include/common/errcode.h

```diff
@@ -39,6 +39,7 @@ static inline std::unordered_map<WasmPhase, std::string> WasmPhaseStr = {
     {WasmPhase::Execution, "execution"}};
 
 /// Error code enumeration class.
+// Remember to keep in sync with `wasmedge.h.in`!
```

Comment:
Can we have a strategy to make error code consistent, 
otherwise, this is prone to forget to be updated.

Reply:
I didn't think too much of it since the other enums are also manually synced currently. I don't know of a good way other than to use a C++ parser or split the error codes out into their own header. The file is already called `errcode.h`, though. What would you suggest? I also prefer not to rely on humans to sync code :)

Reply:
I also haven't found a good way. A little surprise that all `wasmedge.h.in` is maintained manually.
Maybe we can add some checker to check the error code in `errcode.h` and `wasmedge.h.in`.  Some regular expressions may help.

Or, in general, we need to use a script to fill the enum parts of `wasmedge.h.in`.  😏

Reply:
I create an issue for this, #349

Reply:
We can move the discussion there, and figure out a better way to do this.